easybuild/easyconfigs/p/poetry/poetry-1.0.9-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,114 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'PythonBundle'
+
+name = 'poetry'
+version = '1.0.9'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://python-poetry.org/'
+description = "Python packaging and dependency management made easy"
+
+toolchain = {'name': 'foss', 'version': '2020a'}
```

Comment:
@Darkless012 Since `Python` is the only dependency, we can move this to `GCCcore` and just have a single easyconfig that works with both `foss` and `intel`?